### PR TITLE
Feature/cmiml 223 add user

### PR DESCRIPTION
**Pull request by kdanilenka32**

### CODE REVIEW

File: src/components/AddUser/AddUser.tsx [hunk 1]
```diff
@@ -16,48 +17,56 @@ export const AddUser = () => {
 
   const [rows, setRows] = useState<Row[]>([]);
 
-  useEffect(() => {
-    const { getInvitations } = users.thunk;
-    (async () => {
-      const result = await dispatch(getInvitations({ id: id as string }));
+  const getInvitationsHandler = async () => {
+    try {
```
Comment (IhorNesterenko99) [inline src/components/AddUser/AddUser.tsx:21]: async thunk already has try catch inside

File: src/components/AddUser/AddUserForm/AddUserForm.schema.ts [hunk 2]
```diff
@@ -1,20 +1,31 @@
 import * as yup from 'yup';
 import i18n from 'i18n';
+import { Roles } from './AddUserForm.const';
```
Comment (IhorNesterenko99) [inline src/components/AddUser/AddUserForm/AddUserForm.schema.ts:3]: Add space pls at the top

File: src/components/FormComponents/SelectController/SelectController.types.ts [hunk 3]
```diff
@@ -1,4 +1,6 @@
+import { ChangeEvent } from 'react';
 import { TextFieldProps } from '@mui/material/TextField';
+
```
Comment (IhorNesterenko99) [inline src/components/FormComponents/SelectController/SelectController.types.ts:3]: remove space pls

File: src/components/FormComponents/TagsInputController/TagsInputController.types.ts [hunk 4]
```diff
@@ -0,0 +1,11 @@
+import { TextFieldProps } from '@mui/material/TextField';
+
```
Comment (IhorNesterenko99) [inline src/components/FormComponents/TagsInputController/TagsInputController.types.ts:2]: remove space pls

File: src/components/AddUser/AddUser.tsx (hunk 1 above)
Comment (kdanilenka32) [reply src/components/AddUser/AddUser.tsx:21]: let's keep it, if something crash in map

**Review by IhorNesterenko99 — APPROVED** [review]

**Pull request merged**